markdown/ch1.markdown

```diff
@@ -305,7 +313,7 @@ y = np.reshape(y, (1, len(y)))
 print(y)
 ```
 
-In order to do broadcasting, the two arrays have to have the same number of dimensions and the sizes of the dimensions need to match (or be equal to 1).
+In order to do broadcasting, the two arrays have to have the same number of dimensions and the sizes of the dimensions need to match or be equal to 1.
```

Comment:
Yes, but the "None" dimensions are automatically prepended when combining arrays of different dimensions, e.g.:

```ipython
In [2]: x = np.zeros((3, 4))
In [3]: y = np.zeros((4))

In [4]: x + y
Out[4]: 
array([[ 0.,  0.,  0.,  0.],
       [ 0.,  0.,  0.,  0.],
       [ 0.,  0.,  0.,  0.]])
```